src/Notifications/FormSubmittedNotification.php

```diff
@@ -52,4 +52,9 @@ protected function buildMailMessage()
             ->subject('Form ' . $this->formEntry->form->title . ' submitted')
             ->view('fila-cms::notifications.form-submitted', ['entry' => $this->formEntry]);
     }
+
+    public function shouldSend($notifiable)
+    {
+        return filter_var($notifiable->email, FILTER_VALIDATE_EMAIL);
```

Comment:
Does this work with multiple email addresses?  What scenarios have you tested? :)